web/cypress/integration/plugin/create-edit-delete-plugin.spec.js

```diff
@@ -36,14 +41,30 @@ context('Create and Delete Plugin List', () => {
     cy.contains('Create').click();
 
     // add test plugins
-    cy.configurePlugins(this.cases);
+    cy.get('@cases').then((cases) => {
+      cy.configurePlugins(cases);
+    })
+  });
+
+  it('should edit the plugin', () => {
+    cy.visit('/plugin/list');
+    cy.get(domSelector.refresh).click();
+    cy.contains(data.name).should('exist').siblings().contains('Edit').click({
```

Comment:
`refreash` is used to wait route list show right? 
may be this is  repeated  with the assertion in line52